browser/ui/webui/settings/brave_sync_handler.cc

```diff
@@ -347,3 +350,15 @@ void BraveSyncHandler::OnCodeGeneratorResponse(
   qr_code_service_remote_.reset();
   ResolveJavascriptCallback(callback_id, base::Value(data_url));
 }
+
+void BraveSyncHandler::HandleStopClearData(const base::Value::List& args) {
```

Comment:
Is there a reason that we can't reuse `HandleReset`? With this approach, we have to do special handling for `StopAndClear`

Reply:
Just tried `HandleReset` - it also works.
Initially I used `StopAndClear`  because it mimics the manual actions we did when such trouble with the encryption key occurred.

I'll remove it.

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/16117/commits/c150c4f7a5cc3373140ccdf39d2f7fc9794deb8a